pkg/k8shandler/cluster.go

```diff
@@ -75,7 +75,7 @@ func (er *ElasticsearchRequest) CreateOrUpdateElasticsearchCluster() error {
 			return er.UpdateClusterStatus()
 		}
 
-		er.UpdateClusterStatus()
+		return er.UpdateClusterStatus()
```

Comment:
we don't want to return here, we want to continue on so we don't need to wait for the next reconciliation event to then look at upgrades